gradle/verification-metadata.xml

```diff
@@ -289,6 +289,61 @@
             <sha256 value="6af31b34813f2de4e201e1d33d51e0b497180e4d48f139e17b22302ac5c36b2b" origin="Generated by Gradle"/>
          </artifact>
       </component>
+      <component group="com.github.pokkst.tor-binary" name="tor-binary-geoip" version="09d93d2537d14cdab34a8949c5fc81436b14fccc">
+         <artifact name="tor-binary-geoip-09d93d2537d14cdab34a8949c5fc81436b14fccc.jar">
+            <sha256 value="52a1198a78b08a9a2c70efb61a1621b32a1e7326e86e64b1638d3ff63314861f" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
+      <component group="com.github.pokkst.tor-binary" name="tor-binary-linux32" version="09d93d2537d14cdab34a8949c5fc81436b14fccc">
+         <artifact name="tor-binary-linux32-09d93d2537d14cdab34a8949c5fc81436b14fccc.jar">
+            <sha256 value="0e312ebe09930ea2471829ae1a52660a003822670ab4b38086daf192d4868e08" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
+      <component group="com.github.pokkst.tor-binary" name="tor-binary-linux64" version="09d93d2537d14cdab34a8949c5fc81436b14fccc">
+         <artifact name="tor-binary-linux64-09d93d2537d14cdab34a8949c5fc81436b14fccc.jar">
+            <sha256 value="fd86fe12398c1adf227a5a5d1e613cbdc798ec394fa877c2724d58de013b210f" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
+      <component group="com.github.pokkst.tor-binary" name="tor-binary-macos" version="09d93d2537d14cdab34a8949c5fc81436b14fccc">
+         <artifact name="tor-binary-macos-09d93d2537d14cdab34a8949c5fc81436b14fccc.jar">
+            <sha256 value="2becebdf7f81154b004502d65383342647dc3546a4dce62ee84d99d9a0fb8462" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
+      <component group="com.github.pokkst.tor-binary" name="tor-binary-windows" version="09d93d2537d14cdab34a8949c5fc81436b14fccc">
+         <artifact name="tor-binary-windows-09d93d2537d14cdab34a8949c5fc81436b14fccc.jar">
+            <sha256 value="3c1bdaeb73dcdb3074ab4620181749a3b7957f06aad2b9443938480728f4f569" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
+      <component group="com.github.preland.netlayer" name="tor" version="0a992b7281d0b309a3e09386cbf04b2b984a51cb">
+         <artifact name="tor-0a992b7281d0b309a3e09386cbf04b2b984a51cb.jar">
+            <sha256 value="d70fbf41ad3c501113e2f466a0ef264354c44f0c7b6f84d8aebd5cf184eb0022" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
+      <component group="com.github.preland.netlayer" name="tor" version="0d3b9a7ea42ea31a08be2d174c81b3daaf5edda9">
```

Comment:
Should these dependencies be listed twice? i.e. two "tor", "tor.external", "tor.native"?

Reply:
^ Just making sure this wasn't lost. :)